src/main/java/sage/Command/AddCommand.java

```diff
@@ -0,0 +1,97 @@
+package sage.Command;
+
+import sage.List.TaskList;
+import sage.SageException;
+import sage.Storage;
+import sage.Task.DeadlineTask;
+import sage.Task.EventTask;
+import sage.Task.Task;
+import sage.Task.ToDoTask;
+import sage.Ui;
+
+public class AddCommand extends Command {
```

Comment:
Perhaps you can add Javadoc for all classes to give a description of what the class is for!